### PR TITLE
media-sound/pms: address numerous qa issues

### PR DESCRIPTION
**Pull request by Hummer12007**
@gentoo/proxy-maint

### CODE REVIEW

File: media-sound/pms/pms-9999.ebuild [hunk 1]
```diff
@@ -4,40 +4,54 @@
...
+	if [[ ${MERGE_TYPE} != binary ]] && use regex; then
+		if tc-is-gcc && ! version_is_at_least 4.9 $(gcc-version); then
+			die "Clang or GCC >= 4.9 is required for proper regex support"
+		fi
+	fi
+}
+
 src_prepare() {
+	eapply "${FILESDIR}/pms-9999-fix-automagic-dep.patch"
 	eapply_user
```
Comment (doughdemon) [inline media-sound/pms/pms-9999.ebuild]: Better use ``PATCHES`` and ``default``

File: media-sound/pms/pms-9999.ebuild [hunk 2]
```diff
@@ -4,40 +4,54 @@
...
 
 	eautoreconf
 }
 
 src_configure() {
 	# Required for ncurses[tinfo]
-	append-cppflags $(pkg-config --cflags ncursesw)
-	append-libs $(pkg-config --libs ncursesw)
+	append-cppflags $($(tc-getPKG_CONFIG) --cflags ncursesw)
+	append-libs $($(tc-getPKG_CONFIG) --libs ncursesw)
```
Comment (doughdemon) [inline media-sound/pms/pms-9999.ebuild]: ambientsound/pms#38 should make superfluous in future

File: media-sound/pms/pms-9999.ebuild (hunk 1 above)
Comment (Hummer12007) [inline media-sound/pms/pms-9999.ebuild]: I don't think you're right, @doughdemon . Default src_prepare does not invoke eautoreconf.

**Comment by doughdemon** [comment]
> Default src_prepare does not invoke eautoreconf.

That's right. So my suggestion was to use ``PATCHES``, and to do just
``default`` and ``eautoreconf`` in src_prepare(). (This seems to be
the common style for EAPI 6 ebuilds.)

**Comment by Hummer12007** [comment]
Yikes! Something went horribly wrong when I rebased. I'll open another pr.

**Pull request closed without merging**